src/ai/backend/client/func/vfolder.py

```diff
@@ -607,10 +607,7 @@ async def delete_files(self, files: Sequence[Union[str, Path]], recursive: bool
 
     @api_function
     async def list_files(self, path: Union[str, Path] = "."):
-        rqst = Request("GET", "/folders/{}/files".format(self.name))
-        rqst.set_json({
-            "path": path,
-        })
+        rqst = Request("GET", "/folders/{}/files".format(self.name), params={"path": str(path)})
```

Comment:
In the Request class, the params attribute [should only accept mappings where the keys are str and the values are either str or int. ](https://github.com/lablup/backend.ai/blob/5ed63d28a6840b11029e61168baddea4af770438/src/ai/backend/client/request.py#L129C4-L136C54)

Reply:
so I convert `path` to `str`